### DISCUSSION TITLE
Would love to be able to use Costume room Highlighter

### DISCUSSION THREAD

**Discussion by l3vines** [runelite/runelite#18056] [Q&A] [answered]
My bank is always a mess and with a plugin like this i could see really quickly what stuff i can store in my house.
https://runelite.net/plugin-hub/show/costume-room-highlighter

**Comment by Nightfirecat** [comment, marked as answer]
We do not maintain this plugin. There is already an issue on their repo asking them to fix their plugin building: michaelperino/CostumeRoomChecker#5
